### DISCUSSION TITLE
Passkey and old password

### DISCUSSION THREAD

**Discussion by Boca-NeedM** [community/community#124536] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey,

I'm using a passkey to login, but the old (and unsafe) password is still active.
How can i delete the password or the password access.

Kind regards,
Boca

**Comment by davevad93** [comment]
Hi @Boca-NeedM ,

I believe that the passwordless feature on GitHub has not been implemented yet, for more information see this [thread](https://github.com/orgs/community/discussions/67791#discussioncomment-7073092).

**Reply by Boca-NeedM** [reply]
Thanks for your answer.
I agree with his words: "I'm not sure what benefit passkeys provide if someone can still authenticate using less-secure methods."

**Reply by yuu-erp** [reply]
Thanks for your answer.
I agree with his words: "I'm not sure what benefit passkeys provide if someone can still authenticate using less-secure methods."

**Reply by davevad93** [reply]
You're welcome @Boca-NeedM and yes, I totally agree with that. The discussion is from September 2023 and one of the people in charge of authentication stuff said it will take at least 1 year before implementing it, so I imagine it won't happen before 2025.

**Comment by dcaayushd** [comment]
Hey Boca!

Thank you for reaching out to the Community.

Try these General Steps:

1. Check the Account Settings
2. Update Password
3. Contact Support

If you have any further questions, feel free to provide more details about the specific platform or service you're using, and the community will be happy to help you further.

**Reply by samus-aran** [reply]
Hi @dcaayushd , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Boca-NeedM ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
